Tests/SwiftLintFrameworkTests/BaselineTests.swift

```diff
@@ -0,0 +1,225 @@
+@testable import SwiftLintBuiltInRules
+@testable import SwiftLintCore
+import XCTest
+
+private var temporaryFilePath: String {
+    let result = URL(
+        fileURLWithPath: NSTemporaryDirectory(),
+        isDirectory: true
+    ).appendingPathComponent(UUID().uuidString).path
+
+#if os(macOS)
+    return "/private" + result
+#else
+    return result
+#endif
+}
+
+private var sourceFilePath: String = {
+    temporaryFilePath + ".swift"
+}()
+
+final class BaselineTests: XCTestCase {
```

Comment:
Should this be `SwiftLintTestCase`? - I couldn't work out why most tests are, but some tests aren't.

Reply:
`SwiftLintTestCase` only make sure all rules are registered. As violations are created manually in your tests, registering rules don't seem to be needed. It's a convenience not a requirement.